addons/mail_bot/static/src/js/mailbot_service.js

```diff
@@ -60,6 +63,7 @@ var MailBotService =  AbstractService.extend({
      * Called when user either accepts or refuses push notifications.
      */
     removeRequest: function () {
+        localStorage.setItem('odoobot_notification_last_request_time', new Date().getTime());
```

Comment:
same here

Reply:
Maybe add some DB identifier in your local storage key? If you are connected to several DB, the keys may collide.